### PR TITLE
manifests: various cleanups using conditional includes and templates

### PR DESCRIPTION
**Pull request by jlebon**
Just a few cleanups I did to showcase https://github.com/coreos/rpm-ostree/pull/3439.

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ref: fedora/${basearch}/coreos/testing-devel
 include: manifests/fedora-coreos.yaml
 
-releasever: "35"
+releasever: 35
```
Comment (bgilbert) [inline manifest.yaml:5]: Will this break [this line](https://github.com/coreos/fedora-coreos-config/blob/7413ad4049882ed2c873db954d6147d8d43f1af0/ci/overrides.py#L159)?

File: manifests/fedora-coreos.yaml [hunk 2]
```diff
@@ -43,12 +43,6 @@ etc-group-members:
   # https://github.com/projectatomic/rpm-ostree/issues/49
```
Comment (bgilbert) [inline manifests/fedora-coreos.yaml:52]: commit message: `specificied`

File: manifest.yaml [hunk 3]
```diff
@@ -1,16 +1,9 @@
-ref: fedora/${basearch}/coreos/testing-devel
-include: manifests/fedora-coreos.yaml
-
 variables:
+  stream: testing-devel
```
Comment (bgilbert) [inline manifest.yaml:2]: We'll need to update `get_stream()` in `overrides.py`.

File: manifest.yaml (hunk 1 above)
Comment (jlebon) [reply manifest.yaml:5]: Good call. Trying it locally, it seems to work. Looks like the API stringifies it: https://github.com/rpm-software-management/dnf/blob/c68038ff1254c91aa90e1dcb640a933c9194aa98/dnf/conf/config.py#L430

File: manifests/disable-zincati-pinger.yaml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+overlay-layers:
+  - overlay.d/16disable-zincati-pinger
```
Comment (dustymabe) [inline manifests/disable-zincati-pinger.yaml]:
```suggestion
  - overlay.d/16disable-zincati-and-pinger
```

File: manifests/iptables-nft.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# Scripts for opting into staying on iptables-legacy after migration. Remove
+# after the next barrier release.
+ostree-layers:
+  - overlay/35coreos-iptables
+
+# Default to iptables-nft. Otherwise, legacy wins. We can drop this once/if we
+# remove iptables-legacy.
+- |
```
Comment (dustymabe) [inline manifests/iptables-nft.yaml:9]: missing `postprocess:`

**Comment by dustymabe** [comment]
maybe for `manifests: add iptables-nft.yaml for f36+` let's drop the denylist entry for the firewall test.

**Review by dustymabe — DISMISSED** [review]
LGTM - one suggestion.

**Comment by dustymabe** [comment]
I guess we'll need to PR all the branches to update the manifest.yaml too.

**Review by dustymabe — APPROVED** [review]

**Pull request merged**